<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThreeViewsClass</class>
 <widget class="QMainWindow" name="ThreeViewsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ThreeViews</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QVTKWidget" name="qvtkWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>441</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>441</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidget_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>441</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QVTKOpenGLNativeWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>350</y>
      <width>441</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionChoose_DICOM_to_Add"/>
    <addaction name="actionSave_Data"/>
   </widget>
   <widget class="QMenu" name="menu3D_reconstruction">
    <property name="title">
     <string>3D reconstruction</string>
    </property>
    <addaction name="actionSurface_Rendering"/>
    <addaction name="actionVolume_Rendering"/>
   </widget>
   <widget class="QMenu" name="menuMap">
    <property name="title">
     <string>Map</string>
    </property>
    <addaction name="actionDoseMapper"/>
   </widget>
   <widget class="QMenu" name="menuCalculation">
    <property name="title">
     <string>Calculation</string>
    </property>
    <addaction name="actionCalculation_with_OpenMC"/>
   </widget>
   <widget class="QMenu" name="menuVersion">
    <property name="title">
     <string>Version</string>
    </property>
    <addaction name="actionVersion_Info"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu3D_reconstruction"/>
   <addaction name="menuMap"/>
   <addaction name="menuCalculation"/>
   <addaction name="menuVersion"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionChoose_DICOM_to_Add">
   <property name="text">
    <string>Choose DICOM to Add</string>
   </property>
  </action>
  <action name="actionSave_Data">
   <property name="text">
    <string>Save Data</string>
   </property>
  </action>
  <action name="actionSurface_Rendering">
   <property name="text">
    <string>Surface Rendering</string>
   </property>
  </action>
  <action name="actionVolume_Rendering">
   <property name="text">
    <string>Volume Rendering</string>
   </property>
  </action>
  <action name="actionDoseMapper">
   <property name="text">
    <string>DoseMapper</string>
   </property>
  </action>
  <action name="actionCalculation_with_OpenMC">
   <property name="text">
    <string>Calculation with OpenMC</string>
   </property>
  </action>
  <action name="actionVersion_Info">
   <property name="text">
    <string>Version Info</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">qvtkopenglnativewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ThreeViews.qrc"/>
 </resources>
 <connections/>
</ui>
